language/move-stdlib/sources/errors.move

```diff
@@ -0,0 +1,81 @@
+/// This module defines a set of canonical error codes which are optional to use by applications for the
```

Comment:
Do we need this actual module? 
It feels like this could be accomplished just by a doc somewhere, since you aren't actually using the constants? Or do you plan on using them?